### PR TITLE
init for  "Develop with Oracle AI and Database Services: Healthcare (Open Source Version)" workshop WMSID 11485

### PR DESCRIPTION
**Pull request by paulparkinson**
# 11485

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x ] All filenames are lower case (including folders, images, files, etc.)
- [ x] Filenames are descriptive
- [ x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [ x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [ x] Image references in markdown contain an alternate text

### CODE REVIEW

**Review by denisemyrick — CHANGES_REQUESTED** [review]
HI Paul, please update the branch to merge with master. Also I noted OracleHealthAI-FlutterSpringBootStack.png and OracleHealthAI-MERNStack.png need to be updated to be lower case. I'll complete the review once the branch is updated. 

Thanks

**Comment by paulparkinson** [comment]
Thanks @denisemyrick !  Sorry I'm confused as I think it is merging to main (which is master) currently?  I made the modes to lower case OracleHealthAI-FlutterSpringBootStack.png and OracleHealthAI-MERNStack.png

File: oracle-health-ai-open-source/ai-document-flutter-ui/ai-document-flutter-ui.md
Comment (ramumg) [inline oracle-health-ai-open-source/ai-document-flutter-ui/ai-document-flutter-ui.md:1]: Please make sure to check the spelling. According is spelled out wrong.

**Review by ramumg — COMMENTED** [review]
Please make sure you follow the QA

File: oracle-health-ai-open-source/ai-document-flutter-ui/ai-document-flutter-ui.md
Comment (paulparkinson) [reply oracle-health-ai-open-source/ai-document-flutter-ui/ai-document-flutter-ui.md:1]: thanks @ramumg fixed that a few others in the push I just made.

File: oracle-health-ai-open-source/ai-vision-setup/ai-vision-setup.md
Comment (denisemyrick) [inline oracle-health-ai-open-source/ai-vision-setup/ai-vision-setup.md:1]: task 7 skips step 5

**Review by denisemyrick — CHANGES_REQUESTED** [review]

File: oracle-health-ai-open-source/ai-vision-setup/ai-vision-setup.md
Comment (paulparkinson) [reply oracle-health-ai-open-source/ai-vision-setup/ai-vision-setup.md:1]: @denisemyrick got it :) thanks.

**Review by rahul-tasker — APPROVED** [review]

**Review by denisemyrick — APPROVED** [review]

**Pull request merged**